server/src/action/manifest/create.js

```diff
@@ -2,6 +2,67 @@ const AbstractAction = require('../abstract')
 const { body } = require('express-validator/check')
 const { sanitizeBody } = require('express-validator/filter')
 
+/**
+ * @api {post} /api/manifest Create
+ * @apiVersion 1.0.0
+ * @apiGroup Manifest
+ * @apiName ManifestCreate
+ *
+ * @apiDescription Create a manifest to compile
+ *
+ * @apiParam  (Body) {String{1..255}}  name    Name of the manifest
+ * @apiParam  (Body) {String{1..}}  content Content of the manifest
+ *
+ * @apiParamExample {json} Request Body
+ * {
+ *    "name": "bonjour",
+ *    "content": "{{content of a python file manifest}}"
+ * }
+ *
+ * @apiSuccess  {Integer}   id                    Id of the manifest
```

Comment:
```suggestion
 * @apiSuccess  {Integer}   id                    ID of the manifest
```

(This mistake is present a couple more time in the PR)